### PR TITLE
Improve compatibility with respect to Eigen

### PR DESCRIPTION
**Pull request by jcarpent**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -27,31 +27,31 @@ cache:
 
 #robotpkg-pinocchio-14.04-release:
 #  <<: *robotpkg-pinocchio
-#  image: eur0c.laas.fr:5000/stack-of-tasks/pinocchio/pinocchio:14.04
+#  image: memmo.laas.fr:5000/stack-of-tasks/pinocchio/pinocchio:14.04
```
Comment (nim65s) [inline .gitlab-ci.yml]: that's `memmos`

Comment (jcarpent) [reply .gitlab-ci.yml]: Why `s`?

Comment (nim65s) [reply .gitlab-ci.yml]: `memmo.laas.fr` was registered as a domain name for a website before we needed a computer for this project, so we could not name the computer with this name. Then, that's Memmo's `S`erver.

Comment (jcarpent) [reply .gitlab-ci.yml]: Good. Thanks.

**Pull request merged**